### PR TITLE
Add a basic utility that keeps the version param across routes, so we can test different variants of the site.

### PR DESCRIPTION
**Pull request by aaronsn**
Not sure if we want to go in this direction but I figured this is a simple enough bit of code it doesn't hurt to add, if it doesn't turn out to be useful we can take it out.

### CODE REVIEW

File: app/src/utils/useSearchParams.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { useLocation } from "react-router-dom";
+
+function useSearchParams() {
+  // Note: URLSearchParams doesn't support IE, if we keep this code and we want
```
Comment (kkatzen) [inline app/src/utils/useSearchParams.ts:4]: Should we make a note or GitHub Issue about this somewhere to make sure we don't forget about this caveat?

File: app/src/utils/LinkWithStickyParams.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import useSearchParams from "./useSearchParams";
+import { Link } from "react-router-dom";
+
+export const STICKY_VERSION_PARAM = "sv";
+
+function LinkWithStickyParams(props: {
+  to: string;
+  children: React.ReactNode;
+}) {
+  let params = useSearchParams();
```
Comment (kkatzen) [inline app/src/utils/LinkWithStickyParams.tsx:11]: This function is only two lines long- is there a strong benefit to having it in its own separate file?

**Review by kkatzen — APPROVED** [review]
Looks good to me!

File: app/src/utils/useSearchParams.ts (hunk 1 above)
Comment (aaronsn) [reply app/src/utils/useSearchParams.ts:4]: Good call, I made a note on https://github.com/SatcherInstitute/het-frontend/issues/15 which has other stuff about compatibility/polyfills.

File: app/src/utils/LinkWithStickyParams.tsx (hunk 2 above)
Comment (aaronsn) [reply app/src/utils/LinkWithStickyParams.tsx]: Moved both into urlutils file

**Pull request merged**